staynomad/Nomad-Front
# added send verifcation email button to listings tab for unverified hosts

# Limited features of unverified hosts

Closes [#166](https://github.com/staynomad/Nomad-Back/issues/166) on Nomad-Back 

# Summary of change


- Unverified hosts no longer have access to creating listings under the listings tab on the My Account page until they become verified.
- Instead, hosts will be prompted to verify their account, which they can do by clicking the Send Verification Email button.

# Testing/Verification

- Sign in to an unverified host account.
- The listings tab should look like this:
<img width="1210" alt="Screen Shot 2021-08-03 at 10 07 10 PM" src="https://user-images.githubusercontent.com/65259944/128127395-ca238974-bce6-47ab-8630-e49c4e7b2191.png">
- Click the send verification button, and an email should be sent to the email linked with the account.
<img width="894" alt="Screen Shot 2021-08-03 at 10 07 40 PM" src="https://user-images.githubusercontent.com/65259944/128127555-334324a2-b0c6-46dd-8845-792b5e622b92.png">
 - Click on the link provided in the email, which should direct to a verification page.
 - The user should now be verified, and the listings tab should allow access to creating and viewing listings. 
<img width="1210" alt="Screen Shot 2021-08-03 at 10 07 59 PM" src="https://user-images.githubusercontent.com/65259944/128127545-2f989d13-e9d5-44d0-8a4f-0a7c5766d805.png">
